src/plugins/discover/public/application/angular/discover.js

```diff
@@ -821,6 +822,14 @@ function discoverController($element, $route, $scope, $timeout, $window, Promise
       });
   };
 
+  $scope.saveSession = async () => {
+    await data.search.session.save();
```

Comment:
Let's remove for now? Also usage signature is outdated and since this is `js` file this is not validated.